reactors/fcs-tasbe/example/fc.json

```diff
@@ -0,0 +1,105 @@
+{
+	"cytometer_configuration":{
+			"channels": [
+				{
+					"name":                         "FITC-A",
```

Comment:
Minor, but looks like large amounts of whitespace here? I like the JSON structure for the parameters.